Source/Core/VideoCommon/CPMemory.h

```diff
@@ -53,24 +55,44 @@ enum
 };
 
 // Vertex array numbers
-enum
+enum class CPArray : u8
+{
+  Position = 0,
+  Normal = 1,
+  Color0 = 2,
+  Color1 = 3,
+  TexCoord0 = 4,
+  TexCoord1 = 5,
```

Comment:
Adding extra empty lines after 61 and 64 would make it consistent with the separation after line 71